src/main/webapp/views/private-room.html

```diff
@@ -1,11 +1,63 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Private Room</title>
-  </head>
-  <script src="/scripts/private-room-script.js"></script>   
-  <body onload="fetchPrivateRoom()">
-    <div id="player"></div> 
-  </body>
+<html>  
+    <head>    
+        <meta charset="UTF-8">    
+        <title>Private Room</title>    
+        <link rel="stylesheet" href="../stylesheets/private-room.css">  
+    </head>  
+    <script src="/scripts/private-room-script.js"></script>     
+    <body onload="loadIframe(); displayChat()">    
+        <div class="videoContainer">        
+            <iframe id="video-tag"                
+                width="640" height="360"                
+                frameborder="0"                
+                style="border: solid 4px #37474F">        
+            </iframe>    
+        </div>    
+        <div class="chatContainer">    
+            <div class="titleContainer">      
+                <label>Chat</label>    
+            </div>    
+            <div class="msgContainer">        
+                <ul id = "chat-messages" class="msgList">            
+                    <li>                
+                        <div class="msgSender">
+                        </div>                
+                        <div class="msgText">                    
+                            <p class="">Hello this is a test comment.</p> <span class="date sub-text">on March 5th, 2014</span>
```

Comment:
don't include test content within the HTML document itself please.

Reply:
Removed test comments